auth-service/src/main/java/com/artsem/api/authservice/broker/UserRollbackMessage.java

```diff
@@ -0,0 +1,8 @@
+package com.artsem.api.authservice.broker;
+
+import lombok.Builder;
+
+@Builder
+public record UserRollbackMessage(
+        String keycloakId
+) {}
```

Comment:
Следует придерживаться единого стиля расстановки скобок.
```suggestion
@Builder
public record UserRollbackMessage(
        String keycloakId
) {
}
```